src/frameworks/nuxt/index.ts

```diff
@@ -108,7 +108,7 @@ export async function getDevModeHandle(cwd: string) {
 
 export async function getConfig(dir: string): Promise<NuxtOptions> {
   const { loadNuxtConfig } = await relativeRequire(dir, "@nuxt/kit");
-  return await loadNuxtConfig(dir);
+  return await loadNuxtConfig({});
```

Comment:
Can you please apply this change here so we still pass the project directory to `loadNuxtConfig`? 
https://github.com/firebase/firebase-tools/blob/c496f61cb683ce2cbb9f8df4196de79efc690cb6/src/frameworks/nuxt/index.ts#L129-L133

I also have this fix in #7297 but that's not merged yet for other reasons.

Reply:
Great idea @leoortizz 👍 I've just pushed that up